pandas/io/formats/style.py

```diff
@@ -325,9 +325,19 @@ def format_attr(pair):
                                   .format(row=r, col=c)})
             body.append(row_es)
 
+        table_attr = self.table_attributes
```

Comment:
Hmmm, I'm not sure about the option affecting `Styler`. Does anyone else (@jorisvandenbossche, @chris-b1) have thoughts?

Adding classes to the `Styler` is pretty easy, and you're always able to add them since you're generating the Styler. With `DataFrame._repr_html`, you don't have that option.

Reply:
It's true this issue is solved by `df.style.set_table_attributes('class="tex2jax_ignore"')`. One could argue that this PR is unnecessary because the solution is already possible in pandas.

But this solution requires knowledge of how MathJax works – something that I've learnt whilst making this PR. The new option `display.html.use_mathjax` is there to make it easier for users to find the solution.

Reply:
Yeah, I could see the argument for not doing it in `Styler` but I agree the PR is fine, especially where it is off by default, and is sort of an obscure thing to look up.